NASA-AMMOS/aerie-ui
# Plan Import/Export UI Improvements

resolves #1401 

To test import:
1. Create a plan
2. Export the plan via the plans table icons column in the right most column
3. Verify that the plan file is downloaded
4. Click the newly moved "Import" button on the plans page and upload the downloaded plan file from step 3
5. Verify that the input fields are pre-populated with the correct values from the plan file
6. Attempt to upload a non .json file
7. Verify that an error is displayed stating that an incorrect file is selected
8. Attempt to upload a JSON file with invalid JSON
9. Verify that an error is displayed

To test export:
1. Create a plan
2. Navigate to the plan by first selecting the plan from the table and clicking the "Open plan" button that appears on the left of the page in place of the "Create" button
3. Once navigated to the plan page, click on the name of the plan in the nav bar
4. Verify that a new "Export as .json" button is presented
5. Verify that clicking on the new button results in a plan JSON download
